stdlib/REPL/src/docview.jl

```diff
@@ -720,7 +720,19 @@ accessible(mod::Module) =
            map(names, moduleusings(mod))...;
            collect(keys(Base.Docs.keywords))] |> unique |> filtervalid
 
-doc_completions(name) = fuzzysort(name, accessible(Main))
+function doc_completions(name) 
```

Comment:
Ugh, this line had a trailing whitespace

Reply:
Oops